### ISSUE TITLE
How does it work?

### ISSUE THREAD

**Issue by stefanbuck** [tennisonchan/octomerge#2]
Would be good to explain roughly how it works. That would be super helpful for all users.

**Comment by sashashakun** [comment]
Agreed. Also interesting why this tool need access to read and write to my repos?
